wandering-warriors/main.py

```diff
@@ -0,0 +1,26 @@
+from kivy.app import App
+from kivy.uix.widget import Widget
+from kivy.lang import Builder
+
+
+class Abacus(Widget):
+    pass
+
+
+class DrawingInput(Widget):
+    pass
+
+
+class Ledger(Widget):
+    pass
+
+
+class AbacusApp(App):
+    def build(self):
+        Builder.load_file("abacus.kv")
+        app = Abacus()
```

Comment:
I don't think the Abacus widget should be the root of the app. It should probably be a GridLayout which contains the Abacus, Ledger, and ledger input widgets. Their positioning doesn't really matter until we finish the storyboarding

Reply:
You're right, there will need to be some kind of master object to contain everything.  Not sure exactly what that is yet.  Could be GridLayout, but wasn't ready to assert that.